### ISSUE TITLE
bnec doesn't read in a tibble

### ISSUE THREAD

**Issue by gerard-ricardo** [open-AIMS/bayesnec#54]
Produces the error "Error in validate_family(family) : 
  argument family either is not an actual family, or is of incorrect class". 

Solution could be to alert uses to convert to data.frame

**Comment by beckyfisher** [comment]
Great idea Gerard - go ahead and implement that!

**Comment by dbarneche** [comment]
Somehow this got fixed in the current version (67e86f9) of the dev branch of bayesnec.
e.g.

```
library(bayesnec)
data(nec_data)
nec_data <- tibble::as_tibble(nec_data)
exmp_a <- bnec(y ~ crf(x, model = "nec4param"), data = nec_data, chains = 2)
autoplot(exmp_a)
check_chains(exmp_a)
exmp_b <- bnec(y ~ crf(x, model = "ecx4param"), data = nec_data, chains = 2)
exmp_c <- exmp_a + exmp_b
autoplot(exmp_c, all_models = TRUE)
exmp_d <- bnec(y ~ crf(x, model = c("nec4param", "ecx4param")),
               data = nec_data, chains = 2)
exmp_e <- pull_out(exmp_d, "nec4param")
exmp_f <- pull_out(exmp_d, "nec")
exmp_g <- amend(exmp_c, drop = "nec4param")
```
